daemon/daemon_windows.go

```diff
@@ -431,11 +432,7 @@ func initBridgeDriver(controller libnetwork.NetworkController, config *config.Co
 	if config.BridgeConfig.FixedCIDR != "" {
 		subnetPrefix = config.BridgeConfig.FixedCIDR
 	} else {
-		// TP5 doesn't support properly detecting subnet
```

Comment:
Removing this as TP5 is really obsolete

Reply:
Any idea why the original code scoped the use of `defaultNetworkSpace` to < 14360 only?

Reply:
Things were still in development on the Windows side, so I think 14360 was a build where new features became available; that's been a long time back, so I'd have to check git history if there's any info about that